documentation/API_documentation/Carrier_Billing_Payment_API.md

```diff
@@ -0,0 +1,241 @@
+# Overview
+
+<span class="colour" style="color:rgb(0, 0, 0)">The Carrier Billing Payment API provides programmable interface for developers and other users (capabilities consumers) to charge
+ a digital content purchase or subscription to the mobile line invoice.  </span> 
+
+## 1\. Introduction
+
+
+The Carrier Billing Payment API is used by an application to charge an amount on the 
+mobile line. It can be easily integrated and used through this secured API and 
+allows end-users to buy digital content in an easy & secured way. The API provides management of a payment entity and its associated lifecycle. 
+
+The carrier billing Payment API is driven from Open Mobile Alliance standard.
+
+
+## 2\. Quick Start
+
+The usage of the API is based on Payment resource, which can be created (in 1 or 2 steps process), confirmed (for 2-steps process), cancelled (for 2-steps process), queried and retrieved.
+
+
+Before starting to use the API, the developer needs to know about the below specified details:
+
+**Payment service endpoint**
+The URL pointing to the RESTful resource of the payment API.
+As we manage 1-step and 2-steps process we have defined 2 separate tags to explicitly
+distinct them in the swagger. A third tag 'payment' is defined for common operation for
+both process (query & retrieve)
+
+**Authentication**
+Security access keys such as OAuth 2.0 client credentials used by Client applications to invoke the Carrier Billing Payment API.
+
+**1-Step & 2-Steps**
+1-Step process is very simple: The request intent is to charge an amount to the mobile line. When the server receives the request, it will check the user account associated with this line, and, if nothing prevent it, the amount is charged and will be either bill in next invoice or removed from current line credit.

```

Comment:
```suggestion
1-Step process is very simple: The request intent is to charge an amount to the mobile line. When the server receives the request, it will check the user account associated with this line, and, if nothing prevents it, the amount is charged and will be either bill in next invoice or removed from current line credit.

```